multicluster/cmd/multicluster-controller/leader.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
```

Comment:
Antrea import should be in a separate section, don't forget to run `make golangci`, it will help you to find static code issue